<?xml version="1.0"?>
<launch>
   <include file="$(find ramaxxbase)/launch/thrain.launch" />
 
   <node name="relay_imu" pkg="topic_tools" type="relay" args="/spacepoint/imu /robot_0/spacepoint/imu" output="screen" />

   <node name="relay_tf" pkg="topic_tools" type="relay" args="/tf /robot_0/tf" output="screen" />

   <include file="$(find hector_mapping)/launch/mapping_default.launch"/>

   <include file="$(find robot_detection)/launch/master.launch">
       <arg name="master" value="durin" />
   </include>

   <node pkg="robot_detection" type="rerouter" name="rerouter" output="screen">
       <param name="master" type="bool" value="true" />
       <param name="laser_frame" type="string" value="/laser" />
       <param name="target_frame" type="string" value="/map" />
   </node>

   <node name="tf_importer" pkg="robot_detection" type="tf_importer" output="screen" />
   <node name="manual_tf_creator" pkg="robot_detection" type="manual_tf_creator" output="screen">
       <param name="frame/from" value="/map" />
       <param name="frame/to" value="/robot_1/map" />
   </node>

   <include file="$(find robot_detection)/launch/1394cam_marl.launch" />

</launch>
